packages/react-components/react-progress/src/components/ProgressField/ProgressField.tsx

```diff
@@ -7,15 +7,15 @@ import {
   useField_unstable,
 } from '@fluentui/react-field';
 import type { ForwardRefComponent } from '@fluentui/react-utilities';
-import { Progress } from '../../Progress';
+import { ProgressBar } from '../../ProgressBar';
 
-export type ProgressFieldProps = FieldProps<typeof Progress>;
+export type ProgressFieldProps = FieldProps<typeof ProgressBar>;
```

Comment:
Should this be `ProgressBarFieldProps`? And the file name changed as well?

Reply:
We are keeping it as `ProgressField`